recipes/extra-cmake-modules/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+mkdir build
+pushd build
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release     \
+      -Wno-dev \
+      ..
+
+make -j ${CPU_COUNT}
+# no make check
```

Comment:
No `ctest` ?

Reply:
No there isn't any if you look at the cmakelists

Reply:
https://cgit.kde.org/extra-cmake-modules.git/tree/CMakeLists.txt#n102

Reply:
Ahh must have missed it.